x-pack/test_serverless/functional/test_suites/security/config.example.context_awareness.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createTestConfig } from '../../config.base';
+
+export default createTestConfig({
```

Comment:
@elastic/kibana-data-discovery team, I have created a separate config for example profiles, because if example profiles are activated, they override any solution profile that may be available. 

With this approach we are able to keep tests for `example` profiles and `solution` profiles separate. Let me know if you think this approach is not optimal.

Reply:
Done here : https://github.com/elastic/kibana/pull/199818/commits/39a02c40af32359edc294cbe42e663a9bf4bb04b